### PR TITLE
Chinese(simplified) docs

### PR DESCRIPTION
**Pull request by jakecastelli**
## Status
**READY**

## Breaking Changes
 NO

## Description
Translating Docs to Simplified Chinese

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------

## Todos
- [x] Refine the current translation
- [x] Check for spelling or errors

## Impact to Remaining Code Base
This PR will affect: None

## Current Progress:
- [X] README
- [X] Cover page
- [X] Sidebar
- Introduction
  - [X] Getting Started
  - [X] Why Bloc?
- Core Concepts
  - [X] bloc
  - [X] flutter_bloc
- [X] Architecture
- [X] Testing
- [X] Naming Conventions
- [X] FAQs

### CODE REVIEW

**Comment by jakecastelli** [comment]
Hi, @felangel , the intro part has been finished, I will try to frequently revise the docs and provide improvement to the existing work. Meanwhile, I will open separate issues/ PR in order to let other people know my progress and be able to join me.  

Cheers,

Jake

File: docs/zh-cn/_sidebar.md [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
-  - [Why Bloc?](whybloc.md)
-  - Core Concepts
-    - [bloc](coreconcepts.md)
-    - [flutter_bloc](flutterbloccoreconcepts.md)
-  - [Architecture](architecture.md)
-  - [Testing](testing.md)
-  - [Naming Conventions](blocnamingconventions.md)
-  - [FAQs](faqs.md)
+  - [准备开始](zh-cn/gettingstarted.md)
+  - [为什么使用Bloc？](zh-cn/whybloc.md)
```
Comment (felangel) [inline docs/zh-cn/_sidebar.md]:
```suggestion
  - [为什么使用Bloc](zh-cn/whybloc.md)
```

File: docs/zh-cn/_sidebar.md [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
+    - [Github 搜索](flutterangulargithubsearch.md)
 
-- Recipes
+- 配方 (Recipes)
 
   - Flutter
-    - [Show SnackBar](recipesfluttershowsnackbar.md)
-    - [Navigation](recipesflutternavigation.md)
-    - [Bloc Access](recipesflutterblocaccess.md)
+    - [展示提示（SnackBar）](recipesfluttershowsnackbar.md)
```
Comment (felangel) [inline docs/zh-cn/_sidebar.md]:
```suggestion
    - [展示提示（SnackBar](recipesfluttershowsnackbar.md)
```

File: docs/zh-cn/architecture.md [hunk 3]
```diff
@@ -1,46 +1,47 @@
-# Architecture
+# 构架
 
-![Bloc Architecture](assets/bloc_architecture.png)
+![Bloc的构架](assets/bloc_architecture.png)
```
Comment (felangel) [inline docs/zh-cn/architecture.md]:
```suggestion
![Bloc的构架](assets/bloc_architecture.png)
```

File: docs/zh-cn/architecture.md [hunk 4]
```diff
@@ -1,46 +1,47 @@
...
-Using Bloc allows us to separate our application into three layers:
+使用Bloc可以将应用程序分为三层：
 
-- Data
-  - Data Provider
-  - Repository
-- Business Logic
-- Presentation
+-  数据层（Data）
+  - 数据提供者（Data Provider）
```
Comment (felangel) [inline docs/zh-cn/architecture.md]:
```suggestion
  - 数据提供者（Data Provider
```

File: docs/zh-cn/architecture.md [hunk 5]
```diff
@@ -1,46 +1,47 @@
...
+  - 数据源/库（Repository ）
```
Comment (felangel) [inline docs/zh-cn/architecture.md]:
```suggestion
  - 数据源/库（Repository)
```

File: docs/zh-cn/architecture.md [hunk 6]
```diff
@@ -1,46 +1,47 @@
...
+-  业务逻辑（Business Logic）
```
Comment (felangel) [inline docs/zh-cn/architecture.md]:
```suggestion
-  业务逻辑（Business Logic)
```

**Pull request merged**